BenadrylAutomation/src/main/java/com/jnj/pageobjects/BenadrylUsesPage.java

```diff
@@ -0,0 +1,291 @@
+		Action.verifyPageUrl(expectedUrl);
+	}
+
+	/**
+	 * Function to verify headings
+	 * 
+	 * @param expectedHeading pass expected heading
+	 */
+	public void verifyheading(String expectedHeading) {
+		WebElement ele = null;
+		String currentHeading;
+		if (expectedHeading.contains("Product Should I Take")) {
+			ele = this.whichProductHeader;
+		} else if (expectedHeading.contains("How to Use BENADRYL")) {
+			ele = this.howToUseHeader;
+		} else if (expectedHeading.contains("for Cold Symptoms")) {
+			ele = this.forColdHeader;
+		} else if (expectedHeading.contains("for Itchy Skin Symptoms")) {
+			ele = this.itchySkinSympHeader;
+		} else {
+			BaseClass.extentFailLog("Invalid header expected : ", expectedHeading);
+		}
+		Action.scrollIntoCenterUsingJS(driver, ele);
+		expectedHeading = Action.removeNonAlphanumericASCII(expectedHeading);
+		currentHeading = Action.removeNonAlphanumericASCII(ele.getText());
+		Action.printAndAssert(currentHeading, expectedHeading);
+		BaseClass.extentInfoLog("Assertion success for header : " + currentHeading);
+	}
+
+	/**
+	 * Function to verify url from articles
+	 * 
+	 * @param heading     pass heading
+	 * @param article     pass article
+	 * @param readMore    pass read more
+	 * @param expectedUrl pass expected Url
+	 */
+	public void verifyArticles(String heading, String article, String readMore, String expectedUrl) {
+		Action.explicitWait(getSpanHeadings(heading), 30);
+		boolean eleDisplayed = getSpanHeadings(heading).isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Related Content header displayed : ", true);
+			String actualHeading = getSpanHeadings(heading).getText();
+			Action.printAndAssert(actualHeading, heading);
+			Action.explicitWait(articleName(article), 30);
+			String actualArticle = articleName(article).getText();
+			Action.printAndAssert(actualArticle, article);
+			Action.explicitWait(readMoreLink(readMore), 30);
+			Action.performActionwithExtentInfoLog(readMoreLink(readMore), "click",
+					"Clicking on : " + readMoreLink(readMore).getText());
+			Action.verifyPageUrl(expectedUrl);
+		} else
+			extentFailLog("Related Content header displayed : ", false);
+	}
+
+	/**
+	 * Function to verify url from references
+	 * 
+	 * @param heading     pass heading
+	 * @param refText     pass reference Text
+	 * @param link        pass expected link
+	 * @param expectedUrl pass expected Url
+	 */
+	public void verifyReferences(String heading, String refText, String link, String expectedUrl) {
+		String actualHeading = getHeadings(heading).getText();
+		Action.printAndAssert(actualHeading, heading);
+		boolean eleDisplayed = refText(refText).isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Reference point displayed : ", true);
+			String actualRef = refText(refText).getText();
+			extentPassLog("Reference point : ", actualRef);
+			Action.performActionwithExtentInfoLog(refLinks(link), "click", "Clicking on : " + refLinks(link).getText());
```

Comment:
@goutamnaik007 pls remove the empty line

Reply:
Removed @vaibhavNagvekarSJI